ChangeLog

```diff
@@ -20,6 +20,9 @@
  * Removed deprecated Cobyla/TNC.setIgnoreFailure method
  * Removed deprecated OptimizationAlgorithm.setMaximumEvaluationNumber method
  * Removed deprecated OptimizationResult.getEvaluationNumber
+ * Deprecated BayesDistribution in favor of JointByConditioningDistribution
+ * Deprecated ConditionalDistribution in favor of DeconditionedDistribution
+ * Deprecated NegativeBinomial in favor of Polya
```

Comment:
seems like the limit in t_docstring_missing must be incremented because it counts the 3 old classes as undocumented

Reply:
Ok. Where can I change it?

Reply:
in t_docstring_missing.py

Reply:
fixed